lib/active_mocker/mock_creator.rb

```diff
@@ -227,21 +224,20 @@ def scope_methods
     end
 
     module DefinedMethods
-
       def instance_methods
-        class_introspector
-          .get_class
-          .public_instance_methods(false)
-          .sort
-          .map { |m| create_method(m, :instance_method) }
+        class_introspector.
```

Comment:
You may have already thought about this, but I'll just play the devil's advocate to see what you think:

```ruby
class Maybe
  def method_missing(*args)
    self.class.new
  end
end

m = Maybe.new

m.some_line. #you can put comments in
  #this comment is also possible
  some_continuation.
  #and this one too
  some_more()

m.some_line #you can do this comment but
  #this comment is a syntax error
  .some_continuation
  #and this one too
  .some_more()

```

I'll switch them back to start of line if you still prefer that, as I'd rather spend time on code and design concepts than syntax, but just wanted to surface this point about start of line periods.